exercises/xorcism/.meta/description.md

```diff
@@ -49,10 +49,17 @@ These traits will be useful:
 
 - [`AsRef`](https://doc.rust-lang.org/std/convert/trait.AsRef.html)
 - [`Borrow`](https://doc.rust-lang.org/std/borrow/trait.Borrow.html)
-- [`ExactSizeIterator`](https://doc.rust-lang.org/std/iter/trait.ExactSizeIterator.html)
 - [`IntoIterator`](https://doc.rust-lang.org/std/iter/trait.IntoIterator.html)
 - [`Sized`](https://doc.rust-lang.org/std/marker/trait.Sized.html)
 
+## Lifetime of `munge` return value
```

Comment:
This section really should go into a file named `.meta/hints.md`. That way, if we discover a use for additional hints in the future, everything remains appropriately grouped. `configlet` is smart about picking up sections from `description.md`, `hints.md`, and I think a few other places to generate the README.

Reply:
Sorry, I'd misunderstood your previous comments, I'll fix this.